sql/com.snowplowanalytics.amp/amp_id_1.sql

```diff
@@ -0,0 +1,26 @@
+-- AUTO-GENERATED BY igluctl DO NOT EDIT
+-- Generator: igluctl 0.6.0
+-- Generated: 2019-11-28 12:04 UTC
+
+CREATE SCHEMA IF NOT EXISTS atomic;
+
+CREATE TABLE IF NOT EXISTS atomic.com_snowplowanalytics_amp_amp_id_1 (
+    "schema_vendor"  VARCHAR(128)  ENCODE ZSTD NOT NULL,
+    "schema_name"    VARCHAR(128)  ENCODE ZSTD NOT NULL,
+    "schema_format"  VARCHAR(128)  ENCODE ZSTD NOT NULL,
+    "schema_version" VARCHAR(128)  ENCODE ZSTD NOT NULL,
+    "root_id"        CHAR(36)      ENCODE RAW  NOT NULL,
+    "root_tstamp"    TIMESTAMP     ENCODE ZSTD NOT NULL,
+    "ref_root"       VARCHAR(255)  ENCODE ZSTD NOT NULL,
+    "ref_tree"       VARCHAR(1500) ENCODE ZSTD NOT NULL,
+    "ref_parent"     VARCHAR(255)  ENCODE ZSTD NOT NULL,
+    "amp_client_id"  VARCHAR(128)  ENCODE ZSTD NOT NULL,
+    "domain_user_id" VARCHAR(128)  ENCODE ZSTD,
```

Comment:
I've suggested a change in the schema, so that this field matches the `atomic` spelling. But the DDL will have to be regenerated if accepted.